src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/rectangleSelection.scss

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
But not on the very first line. That's only because there is a import at the very beginning.